README.md

```diff
@@ -1 +1,583 @@
+
+Ballerina observability can be categorized into two types as,
+- Default observability - Each Ballerina service can expose its internal information just by enabling the built-in observability. (Eg : Network call duration)
+- User-defined observability - More specific data can be retrieved by using user-defined observability. (Eg : Function execution time) This requires the explicit import of the package `ballerina/observe`.
+
+> This guide is a modification of the [Parallel Service Orchestration guide](https://github.com/ballerina-guides/parallel-service-orchestration) to explain a real-world use case of Ballerina observability.
+
+Following are the sections available in this guide.
+
+- [Introduction](#Introduction)
+- [Prerequisites](#prerequisites)
+- [Testing the services](#Testing_the_services)
+- [Observability](#observability)
+- [Deployment](#deployment)
+
+## Introduction
+
+This guide explains the applicability of Ballerina observability in below scenarios. 
+- Identifying the bottleneck
+- Identifying the unavailble services
+- Determining database call duration and bottleneck queries
+- Function execution time (user-defined observability)
+
+Consider a real-world use case of a travel agency that arranges complete tours for users. A tour package includes airline ticket reservation, hotel room reservation and car rental. Therefore, the travel agency service requires communicating with other necessary back-ends. Those back-ends query their databases to retrieve information of availability of their services.
+
+Travel Agency service communicates with multiple services. Airline reservation service has multiple resources for three different airlines, whereas Hotel reservation and Car rental services only have a single resource. The travel agency service checks all airline resources in parallel to select the cheapest available. Each associated service then queries their database for the availability and information of their services. Only Hotel resevation service does not have a associated database. Rather the response is hard-coded to demonstrate the difference in traces.
+
+![Ballerina Observability Sample](/images/ballerina-observability-sample.png)
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE
+- [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+- [Prometheus](https://ballerina.io/learn/how-to-observe-ballerina-code/#prometheus)
+- [Grafana](https://ballerina.io/learn/how-to-observe-ballerina-code/#grafana)
+- [Jaeger](https://ballerina.io/learn/how-to-observe-ballerina-code/#jaeger-server)
+
+### Optional requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
```

Comment:
Shall we remove Atom from this? We don't maintain this plugin

Reply:
[Regarding the runtime review](https://github.com/ballerina-guides/microservices-observability/pull/1#discussion_r217954478), is runtime publicly available in anywhere? All I see is the BDK (or whatever it is called) not the runtime separately.